freediscovery/server/resources.py

```diff
@@ -846,3 +848,31 @@ def post(self, **args):
         res = model.fe.db.render_dict(scores_pd)
 
         return {'data': res}
+
+
+# ============================================================================ #
+#                            Custom Stop Words
+# ============================================================================ #
+
+
+class CustomStopWordsApi(Resource):
+    @doc(description="Perform a mechanism for adding / managing custom stop words")
+    @use_args({"name" : wfields.Str(required=True),
+               "stop_words" : wfields.List(wfields.Str(), required=True)})
+    @marshal_with(CustomStopWordsSchema())
+    def post(self, **args):
+        self.name = args['name']
+        self.stop_words = args['stop_words']
+        self.model = _StopWordsWrapper(cache_dir=self._cache_dir)
+        self.model.save(name = self.name, stop_words = self.stop_words)
+        return self.model
```

Comment:
Generally all REST API methods should return a dict. In this case, we could for instance return `{'name': name}` and we could modify the `CustomStopWordsSchema` class accordingly (I don't think we need to return `stop_words`).